src/App.tsx

```diff
@@ -1,24 +1,147 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import { TodosList } from './components/TodosList';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { Notifications } from './components/Notifications';
+import { Todo } from './types/Todo';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
+import { Status } from './types/Status';
+import { ErrorTypes } from './types/ErrorTypes';
+import { RequestTodo } from './types/RequestTodo';
 
-const USER_ID = 0;
+const USER_ID = 6325;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [status, setStatus] = useState<Status>(Status.ALL);
+  const [errorType, setErrorType] = useState<ErrorTypes>(ErrorTypes.NONE);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [todoIdsToDelete, setTodoIdisToDelete] = useState<number[]>([]);
+  let visibleTodos = todos;
+
+  const getTodosFromServer = async () => {
+    try {
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch (error) {
+      setErrorType(ErrorTypes.UPLOAD_ERROR);
+    }
+  };
```

Comment:
small optimisation advice: you can move creating of this function inside useEffect(to line 34)